README

```diff
@@ -1,3 +1,50 @@
+AT91 U-Boot Repository
+======================
+
+This is a fork of the U-Boot source code for the AT91 family of ARM
+microprocessors.
+
+https://www.microchip.com/design-centers/32-bit-mpus
+
+This repository is intended to make available a version of U-Boot that works on
+all our supported boards at all times. We try to keep this fork as close as
+possible to mainline U-Boot.
+
+The latest version of this repository can be found at:
+
+https://github.com/linux4sam/u-boot-at91
+
+Since U-Boot is closely related to Linux, please read section 3.1 of Microchip
+AN2772:
+
+"Linux® Basics and Solutions for Microprocessors"
+
+http://ww1.microchip.com/downloads/en/Appnotes/Linux-Basics-and-Solutions-for-Microprocessors-Application-Note-DS00002772A.pdf
+
+When porting U-Boot to a new board design which utilises a Microchip AT91 MPU,
+a typical developer workflow might be:
+
+1. With reference to https://www.linux4sam.org/bin/view/Linux4SAM compile
```

Comment:
https://www.linux4sam.org/ only will be expanded to https://www.linux4sam.org/bin/view/Linux4SAM and is more future proof (if we change wiki engine one day).